package.json

```diff
@@ -19,7 +19,7 @@
     "async-waterfall": "^0.1.5",
     "body-parser": "^1.17.x",
     "cheerio": "~1.0.0-rc.2",
-    "cloudant": "^1.10.0",
+    "cloudant": "^1.10.0-NOTICE",
```

Comment:
whats up with `-NOTICE` ?

Reply:
I think I uninstalled and reinstalled cloudant through npm and that's what added it. Not sure what it does, but we don't need it.